__tests__/news/news.mutation.test.js

```diff
@@ -0,0 +1,260 @@
+              text: [
+                { lang: "uk", value: "u p d a t e N1 d a s d" }
+                { lang: "eng", value: "update dN1 a s d" }
+              ]
+            }
+          ) {
+            ... on News {
+              _id
+              text {
+                lang
+                value
+              }
+              title {
+                lang
+                value
+              }
+              author {
+                name {
+                  lang
+                  value
+                }
+              }
+            }
+            ... on Error {
+              message
+              statusCode
+            }
+          }
+        }
+      `,
+      variables: { id: NEWS_DOES_NOT_EXIST_ID },
+    });
+    expect(res.data.updateNews).toHaveProperty('message');
+    expect(res.data.updateNews).toHaveProperty('statusCode');
+  });
+  test('#4 delete news', async () => {
+    const res = await client.mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          deleteNews(id: $id) {
+            ... on News {
+              text {
+                lang
+                value
+              }
+              author {
+                name {
+                  lang
+                  value
+                }
+              }
+            }
+            ... on Error {
+              statusCode
+              message
+            }
+          }
+        }
+      `,
+      variables: { id: NEWS_ID },
+    });
+    expect(res.data.deleteNews).toBeDefined();
+    expect(res.data.deleteNews).not.toBeNull();
+    expect(res.data.deleteNews.text).toBeInstanceOf(Array);
+    expect(res.data.deleteNews).toHaveProperty('text', [
+      { __typename: 'Language', lang: 'uk', value: 'u p d a t e N1 d a s d' },
+      { __typename: 'Language', lang: 'eng', value: 'update dN1 a s d' },
+    ]);
+    expect(res.data.deleteNews.author).toBeInstanceOf(Object);
+    expect(res.data.deleteNews).toHaveProperty('author', {
+      __typename: 'Author',
+      name: [
+        { __typename: 'Language', lang: 'uk', value: 'updated sd' },
+        { __typename: 'Language', lang: 'eng', value: 'updated sd' },
+      ],
+    });
+  });
+  test('#5 delete not existing news should return error', async () => {
+    const res = await client.mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          deleteNews(id: $id) {
+            ... on News {
+              text {
+                lang
+                value
+              }
+              author {
+                name {
+                  lang
+                  value
+                }
+              }
+            }
+            ... on Error {
+              statusCode
+              message
+            }
+          }
+        }
+      `,
+      variables: { id: NEWS_DOES_NOT_EXIST_ID },
+    });
+    expect(res.data.deleteNews).toBeDefined();
+    expect(res.data.deleteNews).not.toBeNull();
+    expect(res.data.deleteNews).toHaveProperty('statusCode');
```

Comment:
Please, check the actual value of the `message` and `statusCode`, I mean numeric value and the error message text